### DISCUSSION TITLE
隐私错误：你的连接不是专用连接

### DISCUSSION THREAD

**Discussion by HoshiuZ** [hexojs/hexo#5292] [Question / Answer] [answered]
hoshiuz.top 使用加密来保护你的信息。这次，当 Microsoft Edge 尝试连接到 hoshiuz.top 时，网站发回了不正常和不正确的凭据。如果攻击者尝试假冒 hoshiuz.top，或者 WLAN 登录屏幕已中断连接，则可能会发生这种情况。你的信息仍然安全，因为在交换任何数据之前，Microsoft Edge 停止了连接。

你现在无法访问 hoshiuz.top，因为网站使用的是 HSTS。网络错误和攻击通常是暂时的，因此该页面以后可能会恢复正常

打开就显示这个，请问各位大佬该如何解决

**Comment by stevenjoezhang** [comment, marked as answer]
你的网站证书和域名不符合，如果你用的GitHub Pages，需要检查一下设置
